ci/check-commit-format.sh

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
+
 REPO_ROOT=$(git rev-parse --show-toplevel)
+
+"${REPO_ROOT}/ci/update-clang-format"
```

Comment:
Should `clang-format-all.sh` do the same? I run that before committing, so I'd get any errors (pre-commit hook calls check-commit-format.sh so I don't run that manually)